docs/java/hello-world-tutorial.md

```diff
@@ -8,16 +8,17 @@ tags: helloworld, java, sdk, intellij, gradle, tutorial
 <img class="docs-image-centered" src="https://raw.githubusercontent.com/temporalio/documentation-images/main/static/astronaut-hello-java.jpg" />
 
 :::note Tutorial information
+
 - **Level:** ⭐ Temporal beginner
 - **Time:** ⏱️ ~20 minutes
 - **Goals:** 🙌
   - Learn how to set up, build, and test a Temporal application project from scratch using the [Java SDK](https://github.com/temporalio/java-sdk).
   - Become more familiar with core concepts and the application structure.
-:::
+    :::
```

Comment:
Make sure to revert this indent, as it causes the whole page to turn into a note block.